content/dstu2/medication-statement.md

```diff
@@ -11,6 +11,10 @@ title: MedicationStatement | DSTU 2 API
 
 <%= terminology_table(:medication_statement, :dstu2) %>
 
+### Contained Medication Bindings
+
+<%= terminology_table(:medication, :dstu2) %>
```

Comment:
It'll be the same across the Medication resources. Renamed the file to 'contained_medication' though just to be clearer.